project_parent_task_filter/views/project_task.xml

```diff
@@ -20,4 +20,28 @@
             </xpath>
          </field>
      </record>
+
+    <record id="view_task_kanban_subtask" model="ir.ui.view">
+        <field name="name">project.task.kanban</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_kanban"/>
+        <field name="arch" type="xml">
+            <field name="color" position="after">
+                <field name="subtask_count"/>
+                <field name="parent_id"/>
+            </field>
+            <xpath expr="//div[hasclass('oe_kanban_bottom_left')]" position="inside">
+                <div class="pull-right" groups="base.group_user" style="margin-left: 10px;"> 
+                    <t attrs="{'invisible': [('parent_id', '!=', False)]}"
+                        groups="project.group_subtask_project">
+                        <a type="object" name="action_start" style="color:green">
```

Comment:
Does this method list the sub tasks? I can't find it in project.task.

Reply:
No, my fault.
It is deleted now

Reply:
Can you make it clickable so it opens the specific sub tasks?

Reply:
@tarteo done!